src/sentry/tasks/relocation.py

```diff
@@ -131,9 +145,197 @@
 ERR_COMPLETED_INTERNAL = "Internal error during relocation wrap-up."
 
 
-# TODO(getsentry/team-ospo#216): We should split this task in two, one for "small" imports of say
-# <=10MB, and one for large imports >10MB. Then we should limit the number of daily executions of
-# the latter.
+@instrumented_task(
+    name="sentry.relocation.uploading_start",
+    queue="relocation",
+    autoretry_for=(Exception,),
+    max_retries=MAX_FAST_TASK_RETRIES,
+    retry_backoff=RETRY_BACKOFF,
+    retry_backoff_jitter=True,
+    soft_time_limit=FAST_TIME_LIMIT,
```

Comment:
You likely also want `acks_late=True`

Reply:
I've added this, as well as a bit of timeout handling, since (I think? Celery docs were a bit unclear on this) we can now get ourselves into an infinite loop of the task failing and never acking.